docs/how-tos/collect-data.md

```diff
@@ -64,6 +64,8 @@ Now that you have seen that the cameras on your Try Viam rover work, **continue
 If you have a computer that can run `viam-server` but no physical sensor, you can use the [`viam:viam-sensor:telegrafsensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) model which measures computer performance metrics.
 {{% /expand%}}
 
+Make sure to connect your sensor or webcam to your computer.
```

Comment:
I think move this to the bullet about cameras and sensors, before the "no hardware" expander. If it's here, it feels like then the people with no hardware need to do it after they've already done it.